std/typecons.d

```diff
@@ -7143,3 +7143,167 @@ unittest
         ubyte, ubyte, T3, T3,
     );
 }
+
+/**
+Ternary type with three truth values: `true`, `false`, and `unknown`.
```

Comment:
This is the place where you should specify they have the names `yes`, `no`, and `unknown` respectively.